unturned/OpenMod.Unturned.Module.Shared/OpenMod.Unturned.Module.Shared.csproj

```diff
@@ -6,10 +6,6 @@
 
   <ItemGroup>
     <PackageReference Include="Lib.Harmony" Version="2.0.2" />
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net461" Version="1.0.0">
```

Comment:
revert

Reply:
Done, i dont know why but for me it compile without

Reply:
is required for linux compilation support